appveyor.yml

```diff
@@ -66,11 +66,10 @@ test_script:
 
       if ($testsExitCode -ne 0) {exit $testsExitCode}
 
-  - ps: |
-      & coverage erase
+  - coverage erase
+  - del /Q coverage.xml
```

Comment:
The `coverage xml` command line tool creates coverage.xml from the private .coverage file and `codecov` uploads it. This happens 2x, once for unittests and once for integrationtrests. I am removing the file to make sure that these are two independent steps.

unittests.xml and integrationtests.xml have nothing to do with coverage. They are storing extended testing reports.